### DISCUSSION TITLE
question about mixed content

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7079] [Q&A] [answered]
We use the axio protocol to access the HTTP interface in Vue, and when running with cargo tauri dev, the access is normal. But after using the cargo tauri build, it runs and displays mixed content, the page at“ https://tauri.localhost Was loaded over HTTPS, this request was automatically upgraded to HTTPS.
But the requests we access can only be run using the HTTP protocol. How can we use the built application http://tauri.localhost Running?

**Comment by FabianLars** [comment, marked as answer]
See https://github.com/tauri-apps/tauri/issues/3007 => tldr: only option right now is the [localhost plugin](https://github.com/tauri-apps/plugins-workspace/tree/v1/plugins/localhost).
